scripts/scripts.js

```diff
@@ -158,6 +175,22 @@ function buildTOCBlock(main) {
   }
 }
 
+/**
+ * Builds template block and adds to main as sections.
+ * @param {Element} main The container element.
+ * @returns {Promise} Resolves when the template block(s) have
+ *  been loaded.
+ */
+async function buildTemplateBlock(main) {
+  const template = getTemplateName();
+  if (template) {
+    const templateLoader = TEMPLATES[template];
+    if (templateLoader) {
+      await templateLoader(main);
+    }
+  }
```

Comment:
*nitpicking* I always suggest using "guard clauses" to limit code indentation
```suggestion
  if (!template) {
    return;
  }
  const templateLoader = TEMPLATES[template];
  if (!templateLoader) {
    return;
  }
 return templateLoader(main);
}
```

Reply:
I see your point. There's an argument in there for indentation vs multiple return statements (which I usually try to avoid), but I'm happy to follow the project style. I've updated the code as you suggested.